lib/msal-common/src/config/PublicClientSPAConfiguration.ts

```diff
@@ -5,14 +5,14 @@
 import { Configuration, buildConfiguration } from "./Configuration";
 
 /**
- * @type AuthOptions: Use this to configure the auth options in the Configuration object
+ * @type SPAAuthOptions: Use this to configure the auth options in the Configuration object
  *
  *  - clientId                    - Client ID of your app registered with our Application registration portal : https://portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/RegisteredAppsPreview in Microsoft Identity Platform
  *  - authority                   - You can configure a specific authority, defaults to " " or "https://login.microsoftonline.com/common"
  *  - redirectUri                 - The redirect URI of the application, this should be same as the value in the application registration portal.Defaults to `window.location.href`.
  *  - postLogoutRedirectUri       - Used to redirect the user to this location after logout. Defaults to `window.location.href`.
  */
-export type AuthOptions = {
+export type SPAAuthOptions = {
     clientId: string;
     authority?: string;
     redirectUri?: string | (() => string);
```

Comment:
Does Node not use redirectURI or postLogoutRedirectUri?

Reply:
It does. We made a choice of making `redirectUri` a mandatory param in the `request`, hence removed it from `configuration`. `msal-node` does not need `postLogoutRedirectUri`

Reply:
I think those kinds of things need to be discussed as major API changes. I don't think making a separate type for something is a good strategy if the alternative is simply including a configuration option for default. It is going to be more maintenance. Could we try to add these options back and consolidating AuthOptions and SpaAuthOptions?

Reply:
Having redirectUri in the base class pollutes the public API for node, as it's only used in one (out of 6+ flows). We can get rid of AuthOptions altogether, and have separate AuthOptions for browser and for Node